src/components/trade/PriceSuggestions/PriceSuggestionWidget.tsx

```diff
@@ -19,28 +27,41 @@ export const PriceSuggestionWidget: React.FC<Props> = (props) => {
     amount,
     baseToken,
     quoteToken,
+    receiveToken,
+    sellToken,
     isPriceInverted,
     priceInputId,
     priceInverseInputId,
+    wasPriorityAdjusted,
     onSwapPrices,
   } = props
-  const { id: baseTokenId, decimals: baseTokenDecimals } = baseToken
-  const { id: quoteTokenId, decimals: quoteTokenDecimals } = quoteToken
+  const { id: baseTokenId, decimals: baseTokenDecimals } = receiveToken
```

Comment:
Why this change? is not better to express in terms of base/quote the price suggestions?
Also, I don't think this component should have "wasPriorityAdjusted". This component should know only the base/quote and wether the price has been inverted or not.